pkg/resource/logging-config/alloy/logging-config.yaml.template

```
@@ -46,3 +46,7 @@ alloy:
       - name: runlogjournal
         hostPath:
           path: /run/log/journal
+kyvernoPolicyExceptions:
+  enabled: true
+  # Ensure the kyverno policy exceptions are created in the namespace where alloy is installed
+  namespace: null
```

Why? I think they need to bé in the giantswarm Namespace for now
```suggestion
  namespace: null
```

---

You're right this is not needed, removing it.